pkg/universe.dagger.io/x/solomon@dagger.io/spectral/lint.cue

```diff
@@ -105,10 +105,14 @@ import (
 	container: docker.#Run & {
 		// Default container image can be overrided.
 		//   'spectral' must be installed and in the PATH.
-		input:              docker.#Image | *_buildDefaultImage.output
-		_buildDefaultImage: docker.#Pull & {
-			source: "stoplight/spectral"
+		input: docker.#Image
```

Comment:
You could also create a definition `#Image` and do `input: *#Image | docker.#Image`

For instance:

```cue
#Image: {
   docker.#Pull & {
      source: "stoplight/spectral"
   }
}
```

Reply:
changes pushed

Reply:
@TomChv I don't understand this approach. It doesn't work for me:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/3187222/170426580-0146120d-ff94-4345-961e-35dae589194e.png">

Reply:
OH true, that's should be 
```cue
_default: #Image 

input: _base.output
```